### PR TITLE
chore: fix max-width on content

### PR DESCRIPTION
**Pull request by Barsnes**
unspotted style issue when storefront PR was merged.
Also fixes code tags not being styled when nested, for eample in a list

### CODE REVIEW

File: apps/storefront/components/MdxContent/MdxContent.module.css [hunk 1]
```diff
@@ -142,15 +146,26 @@
...
+  /*   & > p,
+  & > h2,
+  & > h3,
+  & > h4,
+  & > ul,
+  & > ol,
+  & > [class~='ds-accordion'],
+  & > [class~='ds-card'] {
+    max-width: 740px;
+  } */
```
Comment (unekinn) [inline apps/storefront/components/MdxContent/MdxContent.module.css:158]: Remove this?

File: apps/storefront/components/MdxContent/MdxContent.module.css [hunk 2]
```diff
@@ -124,6 +124,10 @@
     border-radius: var(--ds-border-radius-md);
   }
 
+  & pre code {
+    background-color: unset;
```
Comment (unekinn) [inline apps/storefront/components/MdxContent/MdxContent.module.css:128]: This looks like it's doing something other than max-width, is the commit message wrong?

**Review by unekinn — COMMENTED** [review]
It might be good to give some screenshots / links to the issues you are fixing

File: apps/storefront/components/MdxContent/MdxContent.module.css (hunk 2 above)
Comment (Barsnes) [reply apps/storefront/components/MdxContent/MdxContent.module.css:128]: yeah this was another small bug I noticed. Can add to the description

File: apps/storefront/components/MdxContent/MdxContent.module.css [hunk 3]
```diff
@@ -142,15 +146,15 @@
...
-    & ol,
-    & [class^='ds-accordion'],
-    & [class^='ds-card'] {
+  & :not([data-unstyled]) {
+    &:is(p),
+    &:is(h2),
+    &:is(h3),
+    &:is(h4),
+    &:is(ul),
+    &:is(ol),
```
Comment (eirikbacker) [inline apps/storefront/components/MdxContent/MdxContent.module.css:155]: Just curious, any preferences on formatting here?
```suggestion
    &:is(p, h2, h3, h4, ul, ol),
```

**Review by eirikbacker — APPROVED** [review]

File: apps/storefront/components/MdxContent/MdxContent.module.css (hunk 3 above)
Comment (Barsnes) [reply apps/storefront/components/MdxContent/MdxContent.module.css:155]: not for me, this was written like this to match the selectors at the bottom, but I don't have a preference for either or

Comment (eirikbacker) [reply apps/storefront/components/MdxContent/MdxContent.module.css:155]: da foreslår jeg slå sammen siden den selectoren kan det <3

**Pull request merged**